README.md

```diff
@@ -0,0 +1,28 @@
+## PyISY
+
+### Python Library for the ISY Controller
+
+This library allows for easy interaction with ISY nodes, programs, program,
+climate module, and network module. This class also allows for functions to be
+assigned as handlers when ISY parameters are changed. ISY parameters can be
+monitored automatically as changes are reported from the device.
+
+### PyISY_beta Fork & PyPi Package
+
+The PyISY_beta package is a drop in replacement & testing package for the original PyISY.  It was created to provide a package to test new modifications while waiting to be incorporated into the original scope.
+
+For more information about the beta package: https://github.com/shbatm/PyISY/tree/PyISY_beta
+
+## Original Repo Information:
+
+For more information, visit the Automicus website.
+
+ORIGINAL AUTHOR: Automicus (Ryan Kraus)
+
+DATE: 4/2014
+
+EMAIL: automicus@gmail.com
+
+WEBSITE: http://automic.us/portfolio/pyisy
+
+GITHUB: https://github.com/automicus/PyISY
```

Comment:
This needs to change to be appropriate for being in the main PyISY repo

Reply:
Fixed in latest commit.